lib/compat/wordpress-5.9/move-theme-editor-menu-item.php

```diff
@@ -5,15 +5,20 @@
  * @package gutenberg
  */
 
-/**
- * Moves the "theme editor" under "tools" in block themes.
+/*
+ * If _add_plugin_file_editor_to_tools is defined, it means the plugin
+ * is running on WordPress 5.9, so no need to change menu location.
  */
-function gutenberg_move_theme_editor_in_block_themes() {
-	if ( ! wp_is_block_theme() || is_multisite() ) {
-		return;
+if ( function_exists( '_add_plugin_file_editor_to_tools' ) ) {
```

Comment:
Should this not be the opposite? 🤔 

```suggestion
if ( ! function_exists( '_add_plugin_file_editor_to_tools' ) ) {
```

We want to move it if the function doesn't exist. If it does, then there's nothing to do... Or am I missing something?

Reply:
No you’re right. Missed this when I was toggling condition for testing 😅 Thanks  🙏